Dockerfile.redhat

```diff
@@ -131,6 +131,10 @@ RUN if [ "$ov_use_binary" == "1" ] && [ "$DLDT_PACKAGE_URL" != "" ]; then true ;
     tar -zxf l_openvino_toolkit*.tgz -C /opt/intel && \
     ln -s /opt/intel/l_openvino_toolkit_runtime_rhel8_p* /opt/intel/openvino
 
+# install opencv
+#
+RUN cd /opt/intel/openvino && curl -s  http://s3.toolbox.iotg.sclab.intel.com/opencv-packages/centos7.tgz | tar -xzf -
```

Comment:
I guess we only need it with `"$ov_use_binary" == "1"` ?

Reply:
it turned out it is needed also for building from ov src